move/its/sources/events.move

```diff
@@ -128,3 +128,50 @@ public(package) fun unregistered_coin_received<T>(
         decimals,
     });
 }
+ 
+// ---------
+// Test Only 
+// ---------
+#[test_only]
+use its::coin::COIN;
+#[test_only]
+use its::token_id;
+
+// -----
+// Tests
+// -----
+#[test]
+fun test_interchain_transfer() {
```

Comment:
This should be two separate tests
`test_interchain_transfer_event_empty_data`
`test_interchain_transfer_event_nonempty_data`

Reply:
done